rust/backend/ebpf/src/sys_sigquit.rs

```diff
@@ -0,0 +1,59 @@
+// SPDX-FileCopyrightText: 2024 Tom Weisshuhn <tom.weisshuhn@fau.de>
+//
+// SPDX-License-Identifier: MIT
+
+use aya_ebpf::{macros::{tracepoint, map}, maps::{RingBuf}, programs::{TracePointContext}, EbpfContext, helpers::gen::bpf_ktime_get_ns};
+use aya_log_ebpf::error;
+use backend_common::{SysSigquitCall};
+
+#[map(name = "SYS_SIGQUIT_EVENTS")]
+pub static SYS_SIGQUIT_EVENTS: RingBuf = RingBuf::pinned(1024, 0);
+
+#[tracepoint]
+pub fn sys_sigquit(ctx: TracePointContext) -> u32 {
+    let pid = ctx.pid();
+    let tid = ctx.tgid();
+
+    let time_stamp: u64;
+    let target_pid: u64;
+    let signal: u64;
+
+
+    unsafe {
+        time_stamp = bpf_ktime_get_ns();
+        target_pid = match ctx.read_at(16) {
+            Ok(arg) => arg,
+            Err(_) => return 1,
+        };
+        signal = match ctx.read_at(24) {
+            Ok(arg) => arg,
+            Err(_) => return 1,
+        };
+    }
+
+    if signal != 3u64 { // libc::SIGQUIT
```

Comment:
libc::SIGQUIT doesn't work (it won't compile) despite my IDE suggesting it and the rust-doc confirming its existence (in the correct library version)

Reply:
the compiler states that there is no SIGQUIT in libc.